core/system/inc/page_allocator_config.h

```diff
@@ -23,18 +23,18 @@
 
 /* We can only protect a small number of pages efficiently, so there should be
  * a relatively low limit to the number of pages.
- * By default a maximum of 16 pages are allowed. This can only be overwritten
- * by the porting engineer for the current platform. */
+ * By default a certain maximum number of pages are allowed. This can only be
+ * overwritten by the porting engineer for the current platform. */
 #ifndef UVISOR_PAGE_MAX_COUNT
-#define UVISOR_PAGE_MAX_COUNT (16UL)
+#define UVISOR_PAGE_MAX_COUNT (128UL)
```

Comment:
@Patater It's not acceptable to increase the pagesize for everybody.
- either we make that value configurable
- or create alternative uVisor-binaries for testing
- or add the page eviction